src/Apps/Order/Components/ExpressCheckout/ExpressCheckoutUI.tsx

```diff
@@ -88,6 +91,7 @@ export const ExpressCheckoutUI = ({
   const [hasMultiplePaymentOptions, setHasMultiplePaymentOptions] =
     useState<boolean>(false)
   const shippingContext = useShippingContext()
+  const { setExpressCheckoutLoaded } = useLoadCheckout()
```

Comment:
We should only call `useLoadCheckout()` once - I just created a ticket for a context for this so I'll do that and we can use that here.